recipes/geocat-datafiles/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "geocat-datafiles" %}
+{% set version = "2022.03.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NCAR/{{ name }}/archive/v{{ version }}.tar.gz # For building from a specific tag's tarball
+  sha256: 68aef597dbe4223fff37b11dab5e8720b1047085af58c95ddfa27d86c5fb75c9
+  # git_tag: v{{ version }} # For building from a specific tag
+  # git_rev: main # For building from branch for debugging purposes
+  # git_url: https://github.com/NCAR/{{ name }}.git
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed -vv .
+  # skip: true  # [py<38 or win]
+
+requirements:
+  build:
+    - python >=3.8                        # [build_platform != target_platform]
+    - pip                                 # [build_platform != target_platform]
+    - setuptools                          # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - pooch                               # [build_platform != target_platform]
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+    - pooch
+  run:
+    - python >=3.8
+    - pooch
+
+test:
+  imports:
+    - geocat.datafiles
+
+about:
+  home: https://geocat.ucar.edu/
+  summary: 'GeoCAT-datafiles is a Python module that allows users to download sample datasets for use with GeoCAT projects.'
+  dev_url: https://github.com/NCAR/{{ name }}
```

Please hardcode the name here
```suggestion
  dev_url: https://github.com/NCAR/geocat-datafiles
```

---

If it causes any issues at all we'll hardcode it. for now, we are maintaining this by hand, and limiting the number of changes we need to make while our packages are on both NCAR's conda channel and conda-forge, should help avoid human error.